lib/dealership.rb

```diff
@@ -0,0 +1,68 @@
+class Dealership
+  attr_accessor :inventory, :name, :location, :lot_value
+
+  def initialize(name, location)
+    @name = name
+    @location = location
+    @inventory = []
+    @lot_value = 0
+  end
+
+  def inventory_count
+    @inventory.length
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    if inventory_count == 0
+      false
+    else
+      true
+    end
+  end
+
+  def cars_by_make(string)
+    cars_with_matching_make = []
+    @inventory.each do |car|
+      if car.make == string
+        cars_with_matching_make << car
+      end
+    end
+    return cars_with_matching_make
+  end
+
+  def total_value
+    @inventory.each do |car|
+      @lot_value += car.total_cost
```

Comment:
Nice creative solution here. Check out the enumerable `.sum` and see if you can refactor this using that one. 😃